modules/structs/repo.go

```diff
@@ -84,23 +90,23 @@ type EditRepoOption struct {
 	// owners and a non-owner tries to change the value of private.
 	Private *bool `json:"private,omitempty"`
 	// Either `true` to enable issues for this repository or `false` to disable them.
-	EnableIssues *bool `json:"enable_issues,omitempty"`
+	HasIssues *bool `json:"has_issues,omitempty"`
```

Comment:
This will break the API?

Reply:
No, this is all new to this PR. It didn't exist before this PR. The only reason it shows it as a change is because I got my stuff into go-sdk before it was merged into the Gitea repo.

Reply:
As I mentioned in my comment after making this commit, I have no idea why I named this "enabled_*" when Github has them as "has_*" and that makes more sense, especially when checking the property of a repo.

Reply:
If `HasIssues` is `nil`, what does it mean? Maybe we should use `bool` but not `*bool`?